docs/concepts/overview.md

```diff
@@ -86,15 +86,15 @@ Switch boot and installation.
 * Actual SONiC installers
 * Miscellaneous: rsyslog/ntp
 
-## Fabric
+## Fabric Agent
```

Comment:
It's "Fabric"

Reply:
Yes, I know the agent is called Fabric. However, on my first read, I thought it would have been clearer to stress that it is an agent, since "fabric" could refer to the HH Fabric. Removing it ...

Reply:
IMO it refers to the HH Fabric here

Reply:
Does it? When I said "HH Fabric" I meant the switches + control nodes ...